client/src/components/BookingCards/BookingCard.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable prettier/prettier */
+import React  from "react";
+import useStyles from "./useStyles";
+import { 
+  Card, 
+  CardHeader, 
+  CardContent, 
+  IconButton, 
+  Typography, 
+  Link,
+  Box, 
+  Grid
+} from "@material-ui/core";
+import { Settings } from "@material-ui/icons";
+
+const mockImg = 'https://badairies.co.uk/assets/admin/plugins/images/users/4.jpg';
+
+interface Props {
+  createDate: any;
+  requests: any;
+  cardAction: boolean;
+  selectedBooking: string;
+  displayCardActions: any;
+  updateSelectedRequest: any;
+}
+
+interface Booking {
+  _id: string;
+  start: string;
+  end: string;
+  accepted: string;
+  declined: string;
+  user: { firstName: string; lastName: string };
+}
+
+function BookingCard({ 
+  createDate, 
+  requests, 
+  cardAction, 
+  selectedBooking, 
+  displayCardActions,
+  updateSelectedRequest}: Props ): JSX.Element {
+  
+  const classes = useStyles();
+
+  return (
+    <Box component="div" style={{height: 264, overflow: "auto"}}>
+      {requests.length 
+        ? requests.map((booking: Booking) => (
```

Comment:
Instead of a tenery like this, we could just do. It's a bit cleaner
```js
{requests.length && . . . }
```

Reply:
I did it that way to display a "No bookings to display" in case of there aren't bookings.
Line 106:
`: <Card>
            <CardContent>
              No bookings to display.
            </CardContent>
          </Card>`